parquet-column/src/main/java/org/apache/parquet/column/statistics/Statistics.java

```diff
@@ -191,6 +191,14 @@ public void mergeStatistics(Statistics stats) {
   abstract public byte[] getMinBytes();
 
   /**
+   * Abstract method to return whether the min and max values fit in the given
+   * size.
+   * @param size a size in bytes
+   * @return true iff the min and max values are less than size bytes
+   */
+  abstract public boolean isSmallerThan(long size);
```

Comment:
size() might be a more intuitive approach and would remove the burden of evaluating this check in the subclasses.